### PR TITLE
feat(domain): add recommended hosting offer

### PR DESCRIPTION
**Pull request by lizardK**
| Question         | Answer
| ---------------- | ---
| Branch?          |  `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-5993
| License          | BSD 3-Clause

## Description

Suggest web hosting promotion

ref: MANAGER-5993

### :bookmark: TODOs

- [x] TRANS-30205 
- [x] TRANS-30241

Signed-off-by: Cyril Biencourt <cyril.biencourt@ovhcloud.com>

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 1]
```diff
@@ -575,5 +575,35 @@
...
+                data-translate="domain_dashboard_recommended_hosting_label"
+                data-translate-values="{ 'reduction': $ctrl.domain.recommendedHosting.reduction, 'type': $ctrl.domain.recommendedHosting.type }"
+            ></p>
+            <a
+                class="oui-link_icon"
+                target="_blank"
+                data-track-on="click"
+                data-track-name="{{ $ctrl.domain.recommendedHosting.tracker }}"
+                data-track-type="navigation"
+                data-ng-href="{{ $ctrl.domain.recommendedHosting.url }}"
```
Comment (JeremyDec) [inline packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html:597]:
```suggestion
                class="oui-link_icon"
                target="_blank"
                data-track-on="click"
                data-track-name="{{ $ctrl.domain.recommendedHosting.tracker }}"
                data-track-type="navigation"
                data-ng-href="{{ $ctrl.domain.recommendedHosting.url }}"
                rel="noopener"
```
As it is external link right ?

**Review by antleblanc — DISMISSED** [review]

**Review by JeremyDec — DISMISSED** [review]

File: packages/manager/apps/web/client/app/domain/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -825,5 +825,9 @@
   "domain_tab_options_delete_error": "Une erreur est survenue lors de la résiliation de l'option.",
   "domain_tab_options_delete_cancel": "Annuler",
   "web_domain_zone_activation_success": "Votre activation a été faite avec succès. Votre produit sera activé dans quelques instants.",
-  "web_domain_zone_activation_order_success": "Votre <a href=\"{{t0}}\" target=\"_blank\">bon de commande</a> a été généré avec succès. Votre produit sera activé lorsque vous l'aurez validé."
+  "web_domain_zone_activation_order_success": "Votre <a href=\"{{t0}}\" target=\"_blank\">bon de commande</a> a été généré avec succès. Votre produit sera activé lorsque vous l'aurez validé.",
+  "domain_dashboard_recommended_hosting_title_Pro": "Hébergement de votre site web",
+  "domain_dashboard_recommended_hosting_title_Perso": "Hébergement de votre blog ou votre site web",
+  "domain_dashboard_recommended_hosting_label": "-{{ reduction }} sur votre Hébergement {{ type }} pour le Black Friday.",
```
Comment (marie-j) [inline packages/manager/apps/web/client/app/domain/translations/Messages_fr_FR.json:831]: nitpick:
```suggestion
  "domain_dashboard_recommended_hosting_label": "-{{ discount }} sur votre Hébergement {{ type }} pour le Black Friday.",
```
would be a more accurate translations imo

**Review by marie-j — APPROVED** [review]

**Comment by antleblanc** [comment]
I'm rebasing on `master` branch.

**Comment by antleblanc** [comment]
Rebased successfully on `master` branch.

**Pull request merged**